### PR TITLE
 Add reference to talk about "Sociotechnical Architecture" of Eduardo…

### LINKED ISSUE #14: Add reference to talk about "Sociotechnical Architecture" of Eduardo da Silva
Hey everyone, 

I watched the talk mentioned in the issue title.
My first thought after this was: very helpful in terms of understanding the steps of the starter modelling process. 

How do you think about adding some kind of reference to this talk in the readme:

https://www.youtube.com/watch?v=ekMPm78KFj0

The introduction into the modellling process starts here:
https://youtu.be/ekMPm78KFj0?t=1820

Best,
Christian

### PR DESCRIPTION
**Pull request by cremich**
Here my PR to add a reference to the mentioned talk of Eduardo.

 closes #14

### CODE REVIEW

**Comment by cakper** [comment]
Looks good to me, thanks for adding it :)

File: README.md [hunk 1]
```diff
@@ -102,7 +102,10 @@ Steps 7 (Define) and 8 (Code) can occur concurrently. This may happen when you a
 
 ## The Process
 
-This process is composed of 8 steps which are introduced below:
+This process is composed of 8 steps which are introduced below.
+
+> A recommended talk about the process is ["Sociotechnical Architecture: co-designing technical & organizational architecture to maximize impact"](https://www.youtube.com/watch?v=ekMPm78KFj0&feature=youtu.be&t=1820) by Eduardo da Silva. He explains the process in the context of typical phases of designing sociotechnical architectures. Eduardo groups the activities in four phases "Align & Understand", "Strategic architecture", "Strategy & Org design" and "Tactical architecture".
```
Comment (emgsilva) [inline README.md:107]: Really good entry (thank you for referring my talk :D). I think we could do a little bit of rewording to improve it even further. Something like: `A recommended talk that gives an overview of the process is .... Furthermore, Eduardo groups the activities of the process (and its 8 steps) in four distinct phases, namely: "1. Align & Understand", "2. Strategic Architecture", "3. Strategy & Org Design" and "4. Tactical Architecture".`

Comment (emgsilva) [reply README.md:107]: Maybe you can also provide a link to the deck - directly to the pages where I give the overview. For example create a link on the "four distinct phases" text to https://speakerdeck.com/emgsilva/intro-to-sociotechnical-architecture-co-designing-technical-and-organizational-architecture-to-maximize-impact?slide=31

Comment (yellowbrickc) [inline README.md:107]: I like the extended version from Eduardo 🙂 
What about this:
```suggestion
> A good talk that gives an overview of the process in the context of typical phases of designing sociotechnical architectures is ["Sociotechnical Architecture: co-designing technical & organizational architecture to maximize impact"](https://www.youtube.com/watch?v=ekMPm78KFj0&feature=youtu.be&t=1820) by [Eduardo da Silva](@emgsilva). Eduardo groups the activities of the process and its 8 steps in four distinct phases, namely: 
1. Align & Understand
2. Strategic Architecture
3. Strategy & Org Design
4. Tactical Architecture.
```

Comment (emgsilva) [reply README.md:107]: Makes it clearer indeed @yellowbrickc, thank you! Additional to this, and as mentioned above, I would include the link to presentation deck into the "in four distinct phases" text (the link can be directly to this part of the presentation, namely: https://speakerdeck.com/emgsilva/intro-to-sociotechnical-architecture-co-designing-technical-and-organizational-architecture-to-maximize-impact?slide=31).

Comment (yellowbrickc) [reply README.md:107]:
```suggestion
> A good talk that gives an overview of the process in the context of typical phases of designing sociotechnical architectures is ["Sociotechnical Architecture: co-designing technical & organizational architecture to maximize impact"](https://www.youtube.com/watch?v=ekMPm78KFj0&feature=youtu.be&t=1820) by [Eduardo da Silva](@emgsilva). Eduardo groups the activities of the process and its 8 steps in [four distinct phases](https://speakerdeck.com/emgsilva/intro-to-sociotechnical-architecture-co-designing-technical-and-organizational-architecture-to-maximize-impact?slide=31), namely: 
1. Align & Understand
2. Strategic Architecture
3. Strategy & Org Design
4. Tactical Architecture.
```

What about this?

Comment (emgsilva) [reply README.md:107]: 👍 exactly, I think this is it! Thank you @yellowbrickc

**Pull request merged**